packages/frontend/src/pages/SavedQueries.tsx

```diff
@@ -23,11 +37,60 @@ const SavedQueries: FC = () => {
 
     return (
         <Page>
-            <SavedQueriesContent
-                savedQueries={data || []}
-                projectUuid={projectUuid}
-                title="All saved charts"
-            />
+            <PageContentWrapper>
+                <PageHeader>
+                    <PageBreadcrumbsWrapper>
+                        <Breadcrumbs2
+                            items={[
+                                {
+                                    href: '/home',
+                                    text: 'Home',
+                                    className: 'home-breadcrumb',
+                                    onClick: (e) => {
+                                        history.push('/home');
+                                    },
+                                },
+                                {
+                                    text: (
+                                        <ResourceBreadcrumbTitle>
+                                            All saved charts
+                                            {savedQueries.length > 0 && (
+                                                <ResourceTag round>
+                                                    {savedQueries.length}
+                                                </ResourceTag>
+                                            )}
+                                        </ResourceBreadcrumbTitle>
+                                    ),
+                                },
+                            ]}
+                        />
+                    </PageBreadcrumbsWrapper>
+
+                    {user.data?.ability?.can('manage', 'Dashboard') &&
```

Comment:
```suggestion
                    {user.data?.ability?.can('manage', 'Dashboard') &&
```

Should be 'SavedChart' instead of Dashboard. 
We should also pass the org and project uuid when checking these permissions. In this case I don't think the buttons will show the users that are only project admins/editors. 🤔 

Eg:
```subject('SavedChart', {
                            organizationUuid: user.data?.organizationUuid,
                            projectUuid,
                        })
```

Reply:
@ZeRego Hmm I see. I just copped this over from the old component. should it be the same with the Dashboards here? 
https://github.com/lightdash/lightdash/blob/feat/resource-table-in-charts-and-dashboards/packages/frontend/src/pages/SavedDashboards.tsx#L88

Reply:
nvm, saw your other comment 👍🏻